hack/kind-cluster-build.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bash
 set -e
 
+ROOT=$(unset CDPATH && cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
+cd $ROOT
```

Comment:
I think we should try not to change the working directory in the shell script, which will make the script readability worse. Users can't know exactly where the file is when they see the relative path. Because he doesn't know if the other place of this script has changed the working directory too.

Reply:
this is the script of this repository, we can assume working directory is at the root of the repo.

Reply:
paths are updated to be absolute: https://github.com/pingcap/tidb-operator/pull/890/commits/972b9f91529dd905fea206f49047b60054c0cb37 

I think changing the working directory is fine because the script belongs to this repository, even if it's execute outside of the repo, it's working directory should be in the repository.